Source/AliveLibAE/Abe.cpp

```diff
@@ -1386,9 +1386,9 @@ void Abe::Update_449DC0()
 
         if (field_114_flags.Get(Flags_114::e114_Bit8_bInvisible))
         {
-            if (!field_170_invisible_timer)
+            if (!field_170_invisible_timer.mTimer)
```

Comment:
instead of !mTimer you could have a bool IsSet() const { return mTimer != 0; } method